bouture.js

```diff
@@ -1,12 +1,43 @@
+const tagsName = ['a', 'abbr', 'address', 'area', 'article', 'aside', 'audio', 'b', 'base', 'bdi', 'bdo', 'blockquote', 'body', 'br', 'button', 'canvas', 'caption', 'cite', 'code', 'col', 'colgroup', 'data', 'datalist', 'dd', 'del', 'details', 'dfn', 'dialog', 'div', 'dl', 'dt', 'em', 'embed', 'fieldset', 'figcaption', 'figure', 'footer', 'form', 'h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'head', 'header', 'hgroup', 'hr', 'html', 'i', 'iframe', 'img', 'input', 'ins', 'kbd', 'label', 'legend', 'li', 'link', 'main', 'map', 'mark', 'meta', 'meter', 'nav', 'noscript', 'object', 'ol', 'optgroup', 'option', 'output', 'p', 'param', 'picture', 'pre', 'progress', 'q', 'rp', 'rt', 'rtc', 'ruby', 's', 'samp', 'script', 'section', 'select', 'slot', 'small', 'source', 'span', 'strong', 'style', 'sub', 'summary', 'sup', 'table', 'tbody', 'td', 'template', 'textarea', 'tfoot', 'th', 'thead', 'time', 'title', 'tr', 'track', 'u', 'ul', 'var', 'video', 'wbr']
+const attributesName = ['accept' , 'accept-charset' , 'accesskey' , 'action' , 'align' , 'alt' , 'async' , 'autocomplete' , 'autofocus' , 'autoplay' , 'bgcolor' , 'border' , 'buffered' , 'challenge' , 'charset' , 'checked' , 'cite' , 'class' , 'code' , 'codebase' , '.class' , 'color' , 'cols' , ' textarea' , 'colspan' , 'content' , 'http-equiv' , 'name' , 'contenteditable' , 'contextmenu' , 'controls' , 'coords' , 'crossorigin' , 'data' , 'data-*' , 'datetime' , 'default' , 'defer' , 'dir' , 'dirname' , 'disabled' , 'download' , 'draggable' , 'dropzone' , 'enctype' , 'POST' , 'for' , 'form' , 'formaction' , 'action' , 'headers' , '<th>' , 'height' , 'hidden' , 'high' , 'href' , 'hreflang' , 'http-equiv' , 'icon' , 'id' , 'integrity' , 'ismap' , 'itemprop' , 'keytype' , 'kind' , 'label' , 'lang' , 'language' , 'list' , 'loop' , 'low' , 'manifest' , 'max' , 'maxlength' , 'minlength' , 'media' , 'method' , 'GET' , 'POST' , 'min' , 'multiple' , 'email' , 'file' , 'muted' , 'name' , 'novalidate' , 'open' , 'optimum' , 'pattern' , 'ping' , 'placeholder' , 'poster' , 'preload' , 'radiogroup' , 'readonly' , 'rel' , 'required' , 'reversed' , 'rows' , 'rowspan' , 'sandbox' , 'scope' , 'scoped' , 'seamless' , 'selected' , 'shape' , 'size' , 'type' , 'text' , 'password' , 'sizes' , 'slot' , 'span' , 'spellcheck' , 'src' , 'srcdoc' , 'srclang' , 'srcset' , 'start' , 'step' , 'style' , 'summary' , 'tabindex' , 'target' , 'title' , 'type' , 'usemap' , 'value' , 'width' , 'wrap']
```

Comment:
I'm skeptical about the idea of listing all attributes, because they're contextual. For instance, `sandbox` only makes sense for `iframe`s.
Maybe there should be a list per HTML element? (+ list of global attributes)
Maybe this can be done as part of a different story.

Reply:
In any case, these elements shouldn't be on the list : 
- `POST`
- `<th>`
- `GET`
- `data-*`

Reply:
I've seen repetitions. Maybe you can a [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) out of the array. This should also make the search below (`attributesName.indexOf(attribute) !== -1`) simpler and more efficient (`attributesName.has(attribute)`)

Maybe this applies to tagsName as well? (should this be in a different story? Does it matter?)

Reply:
For the variable names, I think `tagNames` and `attributeNames` would be more gramatically-correct english. 
Sorry for not catching this the last time. It's fine if this is not fixed

Reply:
I've remove the attributes list, I prefer postpone this job in an other story.